requirements.txt

```diff
@@ -4,6 +4,7 @@ Flask-Admin==1.1.0
 Flask-RESTful==0.3.5
 Flask-Login==0.3.2
 Flask-OAuthLib==0.9.2
+Flask-SQLAlchemy==2.1
```

Comment:
In another project I used [alchy](https://github.com/dgilland/alchy) which also has Flask-Alchy which is a drop-in replacement for it. The benefit is that we won't need Flask session whenever we're using the DB. The downside is that alchy never seemed to gain mind share unlike Flask-SQLAlchemy.

TL;DR: we can keep Flask-SQLA and see if it adds too much boilerplate to the jobs code. If it isn't, keep it. Otherwise consider alchy.